src/client/app/modules/routes/edit-routes.jade

```diff
@@ -13,7 +13,7 @@ form(name="editForm")
     p {{ vm.formInfo.data.length === 0 ? 'No data' : vm.formInfo.data | json }}
 
   div.methods-selector.form-group(ng-hide="vm.formInfo.persistence")
-    div.method.checkbox-inline(ng-repeat="method in vm.allMethods")
+    div.method.checkbox-inline(ng-repeat="method in vm.allMethods" ng-class="method + 'inputBox'")
```

Comment:
change this to be spinal-case, so it'd be like `get-inputbox`

`ng-class="method.toLowerCase() + "-inputbox"` probably will do it